wasm.md

```diff
@@ -1411,7 +1406,8 @@ The `data` initializer simply puts these bytes into the specified memory, starti
          <curModIdx> CUR </curModIdx>
          <moduleInst>
            <modIdx> CUR </modIdx>
-           <memIndices> MEMIDX |-> ADDR </memIndices>
+             <memIds> IDS </memIds>
```

Comment:
syntax. Should `<memIds>` be aligned up with `<modIdx>`?

Reply:
Yep, fixed.